### PR TITLE
 Rename from Google Listings and Ads to Google for WooCommerce

### PR DESCRIPTION
**Pull request by puntope**
### Changes proposed in this Pull Request:

PT: pcTzPl-2i9-p2

This PR replaces all the "Google Listings and Ads" ocurrences with "Google for WooCommerce"

### Detailed test instructions:

1.  Verify in Codebase that there are no instances left of Google Listings and Ads or Google Listings & Ads.
2. Checks (PHP Unit and JS tests) and [E2E tests](https://github.com/woocommerce/google-listings-and-ads/actions/runs/9157439287) are passing.
3. Check the changes in this PR to see if they make sense.

### Changelog entry

> Update - Update extension naming from Google Listings and Ads to Google for WooCommerce

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -525,7 +525,7 @@
 = 1.12.0 - 2022-03-29 =
 * Add - Additional data points for tracker snapshot.
 * Add - Enables merchants to select multiple countries as their audience when creating a Google Ads campaign (Smart Shopping Campaign).
-* Add - Google Listings And Ads product attributes icon.
+* Add - Google for WooCommerce And Ads product attributes icon.
```
Comment (ianlin) [inline changelog.txt]: The `And Ads` wasn't replaced here.

I was actually wondering should we do the rename in changelog? As the changelog records what we've done in the past and the name Google Listings & Ads would be the history as well.

**Review by ianlin — COMMENTED** [review]
Thanks for rebranding it in the plugin code. It looks all good, just found a place that wasn't replaced correctly.

File: changelog.txt (hunk 1 above)
Comment (puntope) [reply changelog.txt]: Right @ianlin I restore the old changes in the changelog and just replaced the title. Can you take another look?

**Review by ianlin — APPROVED** [review]
Thanks for the extra change, LGTM. 👍

**Comment by puntope** [comment]
Closed in favor of https://github.com/woocommerce/google-listings-and-ads/pull/2481

**Pull request closed without merging**